client/secureclient/src/main/java/com/linkedin/openhouse/client/ssl/WebClientFactory.java

```diff
@@ -160,9 +160,6 @@ private WebClient createSecureWebClient(String baseUrl, String token, String tru
     }
     ClientHttpConnector connector = new ReactorClientHttpConnector(client);
     WebClient.Builder webClientBuilder = createWebClientBuilder();
-    if (token != null) {
-      webClientBuilder = webClientBuilder.defaultHeaders(h -> h.setBearerAuth(token));
```

Comment:
We need to remove from line 113 as well as test. I think local docker might be using http url.

Reply:
ApiClient overrides the Authorization header with its default header while preparing for request. SO, I don't think it should matter it's `http` or `https` url. 
However, for consistency, removed setting headers in `http` request case as well.